pkg/windows/windows.go

```diff
@@ -797,15 +799,19 @@ func (vm *windows) newFileInfo(path string) (*payload.FileInfo, error) {
 func (vm *windows) removeHNSNetworks() error {
 	vm.log.Info("removing HNS networks")
 	// VIP HNS endpoint created by the operator is also deleted when the HNS networks are deleted.
-	cmd := "\"$net0 = Get-HnsNetwork | where { $_.Name -eq '" + BaseOVNKubeOverlayNetwork + "'};" +
-		"$net1 = Get-HnsNetwork | where { $_.Name -eq '" + OVNKubeOverlayNetwork + "'};" +
-		"$net0 | Remove-HnsNetwork;" +
-		"$net1 | Remove-HnsNetwork;\""
-	_, err := vm.Run(cmd, true)
-	// PowerShell returns error waiting without exit status or signal error when the HNS networks are removed.
+	baseOvnCmd := "\"Get-HnsNetwork | where { $_.Name -eq '" + BaseOVNKubeOverlayNetwork + "'}| Remove-HnsNetwork;\""
+	_, err := vm.Run(baseOvnCmd, true)
+	if err != nil {
+		return errors.Wrapf(err, "error removing %s required HNS networks", BaseOVNKubeOverlayNetwork)
+	}
+	if err = vm.Reinitialize(); err != nil {
+		return errors.Wrapf(err, "error reinitializing VM after removing %s HNS network", BaseOVNKubeOverlayNetwork)
```

Comment:
s/reinitializing VM/reinitializing ssh client

`reinitializing VM` may lead to misunderstanding, for example: re-configuring/rebooting the VM

Reply:
We are not really  rebooting or reconfiguring the VM, we only [re-initalize the ssh client](https://github.com/openshift/windows-machine-config-operator/blob/374053964b55c49aec644f4398789100c70667ea/pkg/windows/windows.go#L151), the wording is consistent with a call [here](https://github.com/openshift/windows-machine-config-operator/blob/374053964b55c49aec644f4398789100c70667ea/pkg/windows/windows.go#L395) and the error message is expanded in the [vm.Reinitialize()](https://github.com/openshift/windows-machine-config-operator/blob/374053964b55c49aec644f4398789100c70667ea/pkg/windows/windows.go#L267) call.